src/main/java/com/zenjava/javafx/maven/plugin/AbstractJfxToolsMojo.java

```diff
@@ -73,7 +73,7 @@
      * <p>
      * This defaults to 'target/jfx/app' and in most cases there is no real need to change this.
      *
-     * @parameter property="jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
+     * @parameter property="jfx.jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
```

Comment:
Yeah, for the jfxXXX properties there are actually 3 options:
*jfxAppOutputDir (there will be hardly a name clash with other plugins)
*jfx.jfxAppOutputDir (easy to use ; duplicated "jfx")
*jfx.appOutputDir (recommended "jfx." prefix ; no duplicate "jfx" ; you have to know that the property name is different than the actual property)

I don't mind much which pattern is used, but think it's a good idea to make an actual decission!